.cirrus.yml

```diff
@@ -485,63 +500,66 @@ task:
         GCLOUD_SERVICE_ACCOUNT_KEY: ENCRYPTED[f12abe60f5045d619ef4c79b83dd1e0722a0b0b13dbea95fbe334e2db7fffbcd841a5a92da8824848b539a19afe0c9fb]
         SHARD: tests
         SUBSHARD: framework_other
+        GOLDCTL: "$CIRRUS_WORKING_DIR/tmp/depot_tools/goldctl"
+        GOLD_SERVICE_ACCOUNT: ENCRYPTED[3afeea5ac7201151c3d0dc9648862f0462b5e4f55dc600ca8b692319622f7c3eda3d577b1b16cc2ef0311b7314c1c095]
+      goldctl_script: ./dev/bots/download_goldctl.sh
       test_all_script:
         - ulimit -S -n 2048 # https://github.com/flutter/flutter/issues/2976
         - dart --enable-asserts dev/bots/test.dart
-    - name: tests_extras-macos
```

Comment:
@Piinks could you please add the following after `test_all_script` so we can see if there is a time drift?

```yaml
on_failure:
  print_failure_time_script: date
```

Reply:
Done! Thank you!

Reply:
Any update on this @fkorotkov ?

Reply:
@Piinks I though you figure it out and fixed #36103 😪

Looks like there is a drift. I think upgrading the image from xcode-10.1 to xcode-10.2 will fix the drift. Maybe land #36205 first?

Our virtualization provider has an obscure behaviour: VMs expecting the same clock speed as on CPU a VM was build on. Xcode 10.1 images were build on old Mac Minis and the newer images were build on the latest Mac Minis 2018.